scripts/print_track_docstring.py

```diff
@@ -2,7 +2,8 @@
 import importlib
 import itertools
 import types
-
+import sys
+sys.path.append('../mirdata')
```

Comment:
I'm a bit confused why are you editing this file?

Reply:
We initially also got a "ModuleNotFoundError" importing mirdata. We fixed this with the above "hack". Please let us know how this can be circumvented.

Reply:
before merging, can this line be removed? I think the hack is not needed if this is run from the top level directory

Reply:
Yes, it can be removed, no problem.